tests/XML/xenc/ReferenceListTest.php

```diff
@@ -77,19 +49,46 @@ public function setup(): void
     public function testMarshalling(): void
     {
         $referenceList = new ReferenceList(
-            [new DataReference('#Encrypted_DATA_ID', [$this->dataReference])],
-            [new KeyReference('#Encrypted_KEY_ID', [$this->keyReference])]
+            [
+                new DataReference(
+                    '#Encrypted_DATA_ID',
+                    [
+                        new Transforms([
```

Comment:
Uhm, why creating the transforms manually, instead of just doing `Transforms::fromXML()`? That part we should test on its own test for `Transforms`, so there's no need to test it here again.

Reply:
Done!